modules/raft/src/test/java/org/apache/ignite/internal/raft/server/impl/RaftServerImpl.java

```diff
@@ -46,7 +46,7 @@
 
 /**
  * A single node service implementation.
- * @deprecated TODO asch Replace with jraft server https://issues.apache.org/jira/browse/IGNITE-14848
+ * Only for test purposes.
  */
 public class RaftServerImpl implements RaftServer {
```

Comment:
We should use more strict check of using given class in tests only, either by moving it to the test scope or checking whether it's used in test environment, otherwise it might occur that it'll be used in non-test env.